core/templates/dev/head/pages/exploration_player/SupplementalCardDirective.js

```diff
@@ -28,24 +28,31 @@ oppia.directive('supplementalCard', [
         '/pages/exploration_player/' +
         'supplemental_card_directive.html'),
       controller: [
-        '$scope', '$window', 'oppiaPlayerService',
-        'playerPositionService', 'playerTranscriptService',
-        'ExplorationObjectFactory', 'windowDimensionsService',
-        'CONTENT_FOCUS_LABEL_PREFIX', 'TWO_CARD_THRESHOLD_PX',
-        'EVENT_ACTIVE_CARD_CHANGED', 'CONTINUE_BUTTON_FOCUS_LABEL',
+        '$scope', '$timeout', '$window', 'hintManagerService',
+        'oppiaPlayerService', 'playerPositionService',
+        'playerTranscriptService', 'ExplorationObjectFactory',
+        'windowDimensionsService', 'CONTENT_FOCUS_LABEL_PREFIX',
+        'TWO_CARD_THRESHOLD_PX', 'EVENT_ACTIVE_CARD_CHANGED',
+        'CONTINUE_BUTTON_FOCUS_LABEL', 'WAIT_FOR_HINT_MSEC',
         function(
-          $scope, $window, oppiaPlayerService,
-          playerPositionService, playerTranscriptService,
-          ExplorationObjectFactory, windowDimensionsService,
-          CONTENT_FOCUS_LABEL_PREFIX, TWO_CARD_THRESHOLD_PX,
-          EVENT_ACTIVE_CARD_CHANGED, CONTINUE_BUTTON_FOCUS_LABEL) {
+          $scope, $timeout, $window, hintManagerService,
+          oppiaPlayerService, playerPositionService,
+          playerTranscriptService, ExplorationObjectFactory,
+          windowDimensionsService, CONTENT_FOCUS_LABEL_PREFIX,
+          TWO_CARD_THRESHOLD_PX, EVENT_ACTIVE_CARD_CHANGED,
+          CONTINUE_BUTTON_FOCUS_LABEL, WAIT_FOR_HINT_MSEC) {
           var updateActiveCard = function() {
             var index = playerPositionService.getActiveCardIndex();
             if (index === null) {
               return;
             }
             $scope.activeCard = playerTranscriptService.getCard(index);
             $scope.clearHelpCard();
+            hintManagerService.setCurrentHintUsable(false);
```

Comment:
Done, takes care internally in hintManagerService now. (Earlier I had it that way to make sure the hint button is not usable immediately on viewing a card for the first time)